diff --git a/node_modules/@types/json-api-serializer/index.d.ts b/node_modules/@types/json-api-serializer/index.d.ts
index fd6491f..b120428 100755
--- a/node_modules/@types/json-api-serializer/index.d.ts
+++ b/node_modules/@types/json-api-serializer/index.d.ts
@@ -24,6 +24,7 @@ declare namespace JSONAPISerializer {
     links?: LinksObject | LinksCallback | undefined;
     meta?: MetaCallback | Meta | undefined;
     beforeSerialize?: BeforeSerializeCallback | undefined;
+    deserialize?: Function;
   }
 
   type Case = 'kebab-case' | 'snake_case' | 'camelCase';
